hal/src/thumbv7em/clock/v2.rs

```diff
@@ -0,0 +1,1042 @@
+//! # Version 2 of the `clock` module
+//!
+//! ## Overview
+//!
+//! This module provides a simple, ergonomic, and most of all **safe** API to
+//! create and manage the clock tree in ATSAMD5x and E5x devices. It uses
+//! [type-level programming techniques](crate::typelevel) to prevent users from
+//! creating invalid or unsound clocking configurations.
+//!
+//! <p style="background:rgba(255,181,77,0.16);padding:0.75em;">
```

Comment:
does this html actually render? I wasn't aware of cargo doc supporting that.

Reply:
It's more like this markdown is rendered down to HTML and this HTML is just passed through :P But it works nicely in both light mode and dark mode so I think it's ok :)

Reply:
For the record, I shamelessly stole this approach from [this comment](https://github.com/rust-lang/rust/issues/73935#issuecomment-873748906).